doc/tutorial/detection/tutorial-detection-apriltag.doc

```diff
@@ -135,6 +135,16 @@ Then in the while loop, at each iteration we acquire a new image
 
 This new image is then given as input to the AprilTag detector.
 
+\section apriltag_detection_rgbd Improve pose estimation accuracy using a RGB-D camera
+
+When the tag is small in the image or when the tag corners location are extracted poorly, if you analyse the pose of the tag you may experience z-axis flipping. Since the resulting pose is computed from the 4 tag corners location, this behavior is inherent to the planar pose estimation ambiguity, see [Dementhon](http://users.umiacs.umd.edu/~daniel/daniel_papersfordownload/oberkampfDeMenthonPaper.pdf).
```

Comment:
```suggestion
When the tag is small in the image or when the tag corners location are extracted poorly, you may experience z-axis flipping if you analyse carefully the pose of the tag. The pose is computed from the 4 tag corners location, assuming a planar object, and this behavior is inherent to the planar pose estimation ambiguity, see [Dementhon](http://users.umiacs.umd.edu/~daniel/daniel_papersfordownload/oberkampfDeMenthonPaper.pdf).
```

Reply:
Maybe add an image in the  tutorial doc to illustrate the issue on a real use case?